### PR TITLE
Fix E2E test report upload URL

### PR DESCRIPTION
**Pull request by mvitale1989**
#### Summary

This PR fixes a bug in the E2E test report upload script, which caused the report to be inaccessible for Windows E2E tests.

#### Ticket Link

https://mattermost.atlassian.net/browse/CLD-5909

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
Approving to unblock when this is good to merge.

**Comment by mvitale1989** [comment]
I also manually launched a [windows pipeline](https://git.internal.mattermost.com/qa/desktop-e2e-testing/-/pipelines/348989), to verify that the produced report links are valid

**Comment by yasserfaraazkhan** [comment]
@mvitale1989  

https://community.mattermost.com/core/pl/kyh1us7syjr19m9tnsu7oqoxzw

maybe the report is not building. It's accessible so this fix is working.

https://mattermost-cypress-report.s3.amazonaws.com/348970-desktop-pr-windows/amd64-cld-5909-fix-e2etest-report-abd9e6c/mochawesome.html

**Comment by mvitale1989** [comment]
That report is only partially working, but it should be fixed by this commit that i already pushed: https://github.com/mattermost/desktop/pull/2776/commits/c1c2dfe6515ced074408126b1394f87c8b1dbc00

tldr: the `replace()` function only substituted one backslash; so e.g. `amd64-cld-5909-fix-e2etest-report-abd9e6c\mochawesome.html` gets properly fixed and you can see it loads, but e.g. `amd64-cld-5909-fix-e2etest-report-abd9e6c\assets\app.css` has more than one backslash, so it's still broken after using `replace()`. Hopefully `replaceAll()` fixes this, waiting for the Windows pipeline to finish, to verify this

**Comment by yasserfaraazkhan** [comment]
thank you Mario

**Review by yasserfaraazkhan — APPROVED** [review]
